core/events/workspace_events.js

```diff
@@ -13,22 +13,22 @@
 goog.module('Blockly.Events.FinishedLoading');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.Events');
-goog.require('Blockly.Events.Abstract');
-goog.require('Blockly.registry');
-goog.require('Blockly.utils.object');
-
-goog.requireType('Blockly.Workspace');
+const Abstract = goog.require('Blockly.Events.Abstract');
```

Comment:
I'm not sure if this is style-guide legal but `@extends {Abstract}` is a bit confusing. I might remove this require and use `Events.Abstract` instead.

Reply:
Went for EventsAbstract, which is allowed for disambiguation/clarity (and which I quite agree is useful!)